### PR TITLE
feat: add mechanism in initramfs to create a symbolic link

### PR DESCRIPTION
**Pull request by JoergZeidler**

### CODE REVIEW

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+fs_link_enabled() {
+    return 0
+}
+
+run_fs_link() {
+
+    FS_LINK_FILE="/rootfs/etc/omnect/create-fs-links.txt"
+    msg "omnect-os initramfs fs link"
+
+    if ls ${FS_LINK_FILE} 1> /dev/null 2>&1; then
```
Comment (HarryWaschkeit) [inline recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link:12]: could be replaced by a simple ...

```suggestion
    if [ -r "${FS_LINK_FILE}" ]; then
```

Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link]: done

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+fs_link_enabled() {
+    return 0
+}
+
+run_fs_link() {
+
+    FS_LINK_FILE="/rootfs/etc/omnect/create-fs-links.txt"
```
Comment (mlilien) [inline recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link:9]: as far as i understand the file will persist and thus this script recreates links on every boot.

Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link:9]: will refactor that part to create the link just once

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        done < ${FS_LINK_FILE}
+    fi
+    return 0
+
+}
+
+fs_link_run() {
+    add_module_post_hook on_exit
+    run_fs_link
+    return=${?}
```
[the remark this answers is not in the document]
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link:27]: code refactored

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link (hunk 2 above)
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-link:9]: done

**Review by mlilien — DISMISSED** [review]

**Review by HarryWaschkeit — APPROVED** [review]

**Review by HarryWaschkeit — APPROVED** [review]

**Pull request merged**